jerry-ext/include/jerryscript-ext/debugger.h

```diff
@@ -31,6 +31,11 @@ void jerryx_debugger_after_connect (bool success);
  */
 bool jerryx_debugger_tcp_create (uint16_t port);
 
+/*
+ * Message transmission interfaces.
+ */
```

Comment:
Don't repeat this comment. Just add the new declaration. The comment marks the beginning of a list. Only this list was one element long, until now.